app/uk/gov/hmrc/agentclientrelationships/connectors/GovernmentGatewayProxyConnector.scala

```diff
@@ -93,7 +93,7 @@ class GovernmentGatewayProxyConnector @Inject()(@Named("government-gateway-proxy
     </GsoAdminGetCredentialsForDirectEnrolmentsXmlInput>.toString()
 
   private def GsoAdminGetUserDetailsXmlInput(credentialIdentifier: String): String =
-    <GsoAdminGetUserDetailsXmlInput xmlns="urn:GSO-SystemServices:external:2.13.3:GsoAdminGetUserDetailsXmlInput">
+    <GsoAdminGetUserDetailsXmlInput xmlns="urn:GSO-System-Services:external:2.13.3:GsoAdminGetUserDetailsXmlInput">
```

Comment:
this is `xmlns="urn:GSO-System- Services:external:2.14.4:GsoAdminGetUserDetailsXmlOutput"` in the doc I'm looking at (GGTE017 v11.0 GG Developer Guide Part 4 - Security Token Service and SOAP Interfaces.pdf), although I don't think the difference is important it's just `2.13.3` vs `2.14.14`